integrations/docker/images/chip-build-imx/Dockerfile

```
@@ -0,0 +1,17 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION} as build
+
```

Make sure that you have `wget` tool

```suggestion
RUN set -x \
    && apt-get update \
    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --no-install-recommends \
    wget=1.20.3-1ubuntu2 \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/ \
    && : # last line
```

---

Hi, Thank you for the comments.
Added the suggested lines into the Docker file to make sure wget available.